orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -363,24 +365,18 @@ class OperationsController {
     }
   }
 
-  private Map<String, String> startPipeline(Map config) {
+  private String startPipeline(Map config) {
     injectPipelineOrigin(config)
     def json = objectMapper.writeValueAsString(config)
-    log.info('requested pipeline: {}', json)
-
     def pipeline = executionLauncher.start(PIPELINE, json)
-
-    [ref: "/pipelines/${pipeline.id}".toString()]
+    return pipeline.id
   }
 
-  private Map<String, String> markPipelineFailed(Map config, Exception e) {
+  private String markPipelineFailed(Map config, Exception e) {
     injectPipelineOrigin(config)
     def json = objectMapper.writeValueAsString(config)
-    log.warn('requested pipeline marked as failed: {}', json)
-
     def pipeline = executionLauncher.fail(PIPELINE, json, e)
-
-    [ref: "/pipelines/${pipeline.id}".toString()]
+    return pipeline.id
```

Comment:
The previous return value from this method was being ignored as the calling code immediately throws an exception.